cpp/src/Slice/GrammarUtil.h

```diff
@@ -193,6 +195,19 @@ class TaggedDefTok : public GrammarBase
     std::string name;
     bool isTagged;
     int tag;
+    StringList metadata;
+};
+
+// ----------------------------------------------------------------------
+// TaggedDefListTok
+// ----------------------------------------------------------------------
+
+class TaggedDefListTok : public GrammarBase
+{
+public:
+
+    TaggedDefListTok() { }
```

Comment:
Why not use the default ctor?

Reply:
Just for consistency, if you look through the rest of the file, we make a habit of declaring the default constructor. I figured it was some weird style convention and have continued to follow it. Otherwise there is no reason for it.

Reply:
Talked with Bernard, he agreed these are useless. I'll remove all the default constructors.